internal/system/agent/logger/logs.go

```diff
@@ -25,4 +25,8 @@ var LoggingClient logger.LoggingClient
 
 func BuildLoggingClient(config *interfaces.ConfigurationStruct, logTarget string) {
 	LoggingClient = logger.NewClient(internal.SystemManagementAgentServiceKey, config.EnableRemoteLogging, logTarget, config.LoggingLevel)
+}
+
+func BuildDefaultClient() {
```

Comment:
This function can be eliminated if telemetry is moved into its own package and out of system/agent